qemu/tests/cfg/rh_qemu_update.cfg

```diff
@@ -0,0 +1,13 @@
+- rh_qemu_update:
+    type = rh_qemu_update
+    only RHEL
+    only x86_64 aarch64
+    only q35
```

Comment:
Looks like we need to correct this as the following.
```
x86_64:
    only q35
```

Reply:
Thanks, that would address the problem. I was thinking about dropping it at all because there is no real dependency on arch or machine type (and someone maybe could want to use this test case on ARM). We can put this into our internal configuration if we wish anyway.

Reply:
+1,  from test case itself, no limitation for arch, machine type and so on.  `only RHEL`  is enough, thanks!